packages/app-project/next.config.js

```diff
@@ -57,14 +57,7 @@ const nextConfig = {
 
   experimental: {
     forceSwcTransforms: true,
-    modularizeImports: {
-      lodash: {
-        transform: 'lodash/{{member}}',
-      },
-      '@zooniverse/react-components': {
-        transform: '@zooniverse/react-components/{{member}}'
-      }
-    },
+    optimizePackageImports: ['@zooniverse/react-components', 'grommet', 'grommet-icons']
```

Comment:
Is `optimizePackageImports` the updated syntax for `modularizeImports`? Why not include `lodash` in the new array?

Reply:
`lodash` is included by default. 

https://nextjs.org/docs/app/api-reference/next-config-js/optimizePackageImports

Reply:
This blog post explains how Next.js handles barrel files and goes into details. 

https://vercel.com/blog/how-we-optimized-package-imports-in-next-js

One takeaway I got from that is that it would probably make sense to remove barrel files from the libraries. They slow down imports and make the bundle bigger than it should be.

Reply:
I was wondering about this too! There's lots of barrel imports in lib-classifier for example.

Reply:
Yeah, I used them a lot, in the classifier, to make code easier to organise but they may have been a bad idea.